columnar/ColumnarFiles.java

```diff
@@ -0,0 +1,85 @@
+package columnar;
+
+import global.AttrType;
+import global.RID;
+import heap.*;
+
+import java.io.IOException;
+
+public class ColumnarFiles {
+
+    private final String name;
+
+    private final Heapfile tidFile;
+
+    private final Heapfile deleteFile;
+
+    private final Heapfile headerFile;
+
+    private final ColumnFile[] columnFiles;
+    private final  int numColumns;
+
+    public ColumnarFiles(
+            String name,
+            String[] columnNames,
+            AttrType[] attrTypes,
+            int numColumns
```

Comment:
@kunalmanishdesai Is `numColumns` required in the constructor? We can just check the length of `columnNames`, right?

Reply:
It looks like an extra step to get num of cols